examples/nemo/eg3/Makefile

```diff
@@ -72,7 +72,7 @@ dl_timer:
 transform: kernels managed_mem
 
 kernels:
-	${PSYCLONE} -l -api "nemo" -s ./kernels_trans.py -opsy psy.f90 \
+	${PSYCLONE} -l all -api "nemo" -s ./kernels_trans.py -opsy psy.f90 \
```

Comment:
As before. If it is needed then we should explain why. If it is not we should remove it.

Reply:
This one is needed so I've added a comment to the Makefile and the README.

Reply:
Great. Modified the copyright notice in the README file.